cupy_setup_build.py

```diff
@@ -558,8 +575,6 @@ def make_extensions(options, compiler, use_cython):
         link_args = s.setdefault('extra_link_args', [])
 
         if module['name'] == 'cusolver':
-            compile_args = s.setdefault('extra_compile_args', [])
-            link_args = s.setdefault('extra_link_args', [])
```

Comment:
Not sure these two lines are here, but they're redundant as we just did this 3 lines above.